e2e/test_sanity.py

```diff
@@ -0,0 +1,42 @@
+from playwright.sync_api import Page, expect
+
+import os
+
+BASE_URL = os.environ.get("BASE_URL", "http://localhost:8080/")
+
+
+def test_ingest_odt(page: Page) -> None:
+    """
+    Creates an investigation, uploads an odt file and awaits it to be there.
+    Searches for a word inside it, opens the file preview, looks at the extracted
+    text and browses to the last page.
+    """
+
+    page.goto(BASE_URL)
+    page.get_by_role("menuitem", name="Investigations").click()
+    page.get_by_role("button", name="New investigation").click()
+    page.get_by_placeholder("Untitled investigation").click()
+    page.get_by_placeholder("Untitled investigation").fill("[test] Don't look in here")
+    page.get_by_role("button", name="Save").click()
+    page.get_by_role("button", name="Upload documents").click()
+    page.get_by_label("Choose files to upload...").set_input_files(
+        "e2e/fixtures/random.odt"
+    )
+    page.get_by_role("button", name="Upload", exact=True).click()
+    page.locator("section").get_by_role("button", name="Close").click()
```

Comment:
Something like this may be a little more robust in case the markup changes or there are other `section` elements:

```suggestion
    page.get_by_role("dialog", name="Upload documents").get_by_role("button", name="Close").click()
```